app/environments/brassbirmingham/brassbirmingham/envs/test.py

```diff
@@ -17,9 +21,17 @@ def resetGame(self, numPlayers):
 
         if numPlayers > 2:
             self.p3 = Player("Sam", self.board)
+        
+            self.board.tradePosts[0].beerAmount
         if numPlayers > 3:
             self.p4 = Player("Mr. Mcdonald", self.board)
 
+        # randomize merchant tile init
```

Comment:
Do we want randomization in our unit tests?

Reply:
What is general practice?

Reply:
If the actual game code has randomization, then we can take that into account in our unit tests, but I don't think you usually add randomization in unit tests? Based on this SO post: https://stackoverflow.com/questions/32458/random-data-in-unit-tests , I guess it's not entirely wrong, but I'm not sure it falls under traditional unit testing

Reply:
Either is fine with me, up to you

Reply:
I removed the randomization